frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/block-pool.ts

```diff
@@ -0,0 +1,105 @@
+import { NS } from '../utils/consts';
+import { commonFlows } from './common';
+import { POOL_PROGRESS } from '../../src/constants/storage-pool-const';
+
+// Pool var
+export const poolName: string = 'example.pool';
+export const replicaCount: string = '2';
+export const volumeType: string = 'ssd';
+
+export const poolMessage: {
+  [key in POOL_PROGRESS]?: string;
+} = {
+  [POOL_PROGRESS.FAILED]: `Pool "${poolName}" already exists`,
+  [POOL_PROGRESS.CREATED]: `Pool ${poolName} was successfully created`,
+  [POOL_PROGRESS.NOTALLOWED]:
+    "Pool management tasks are not supported for default pool and Openshift Container Storage's external mode.",
+};
+
+export const navigateToBlockPool = () => {
```

Comment:
```suggestion
const navigateToBlockPool = () => {
```
I don't see it being to used anywhere else.